src/model/data_adapter.py

```diff
@@ -3,7 +3,7 @@
 from pymongo.mongo_client import MongoClient
 from pymongo.server_api import ServerApi
 
-MONGODB_URI = ""
+MONGODB_URI = "mongodb+srv://guest1:guest1@cluster0.kk0nx8e.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
You shouldn't upload your file with URI since it is the way to access your database